buddyboss/buddyboss-platform
# PROD-810 Document visibility level apply_filter missing REST endpoint - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2463 .

### How to test the changes in this Pull Request:

1. Add a custom filter for 'bp_document_get_visibility_levels' filter hook
2. After that see the response for this endpoint: {{root_url}}/wp-json/buddyboss/v1/document/details
3. The custom document visibility level filter should work.

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Added document level filter for document details REST endpoint
